app/web_hub.go

```diff
@@ -356,22 +359,31 @@ func (h *Hub) Register(webConn *WebConn) {
 func (h *Hub) Unregister(webConn *WebConn) {
 	select {
 	case h.unregister <- webConn:
-	case <-h.stop:
```

Comment:
`stop` vs. `didStop` probably makes no semantic difference, but I think it makes sense to deny callers only when the web hub has actually stopped vs. having been requested to stop, since it avoids coupling the two concepts. This variable is probably better called `h.done` as such.